arlo_server/routes.py

```diff
@@ -865,12 +872,11 @@ def ballot_set(election_id, jurisdiction_id, batch_id, ballot_position):
         )
 
     ballot = ballots[0]
-
-    if "vote" in attributes:
-        ballot.vote = attributes["vote"]
-
-    if "comment" in attributes:
-        ballot.comment = attributes["comment"]
+    ballot.interpretations = [
+        deserialize_interpretation(ballot.id, interpretation)
+        for interpretation in attributes["interpretations"]
+    ]
```

Comment:
adding `delete-orphan` in the relationship config allows us to do the update this way. any `Interpretation` that was in the old array but not the new array will be orphaned and deleted automagically